### PR TITLE
add filtering to backtest page

### PR DESCRIPTION
**Pull request by AntonNazarenko**
added market filtering ability to instead of searching with scrolling

### CODE REVIEW

File: src/components/Backtester/style.scss [hunk 1]
```diff
@@ -75,4 +82,4 @@
 
 .hfui-backtester__wrapper {
   padding: 20px;
-}
+}
```
Comment (robertkowalski) [inline src/components/Backtester/style.scss:85]: missing newline at end of file

Comment (robertkowalski) [reply src/components/Backtester/style.scss:85]: this issue comes up sometimes, can you set your editor to add a newline on each saving automatically?

Comment (AntonNazarenko) [reply src/components/Backtester/style.scss]: Fixed

File: src/components/Backtester/forms/HistoricalForm.js [hunk 2]
```diff
@@ -11,6 +12,7 @@ import Checkbox from '../../../ui/Checkbox/Checkbox'
 const ONE_MIN = 1000 * 60
 const ONE_HOUR = ONE_MIN * 60
 const ONE_DAY = ONE_HOUR * 24
+const MAX_DATE = new Date()
```
Comment (robertkowalski) [inline src/components/Backtester/forms/HistoricalForm.js:15]: hm, did you add other code? `const MAX_DATE = new Date()`

Comment (robertkowalski) [reply src/components/Backtester/forms/HistoricalForm.js:15]: https://github.com/bitfinexcom/bfx-hf-ui/pull/258/files doesn't have it

Comment (AntonNazarenko) [reply src/components/Backtester/forms/HistoricalForm.js:15]: my branch was outdated, the master branch already contains this code ( it's added by Alex )

Comment (AntonNazarenko) [reply src/components/Backtester/forms/HistoricalForm.js:15]: Should i remove it, or left as is, what do you think? @robertkowalski

Comment (robertkowalski) [reply src/components/Backtester/forms/HistoricalForm.js:15]: can you rebase please?

**Pull request closed without merging**